### PR TITLE
First and Last Moments of Temporal Regions

### PR DESCRIPTION
**Pull request by gregfowlerphd**

### CODE REVIEW

File: design-patterns/time/CCO DP, p. 17.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+graph TD
```
Comment (giacomodecolle) [inline design-patterns/time/CCO DP, p. 17.md:1]: change to BT

File: design-patterns/time/CCO DP, p. 17.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    A{Temporal Region 1} -->|rdf:type| B[Temporal Region BFO_0000008]
+    C{Temporal Instant 1} -->|first instant of BFO_0000221| A
+    D{Temporal Instant 2} -->|last instant of 
+    BFO_0000223| A
+    C -->|precedes 
+    BFO_0000063| D
+    C -->|rdf:type| E[Temporal Instant BFO_0000063]
+    D -->|rdf:type| E
+
+    classDef yellow fill:#EDF2AE
```
Comment (giacomodecolle) [inline design-patterns/time/CCO DP, p. 17.md:11]: change yellow code

**Review by giacomodecolle — CHANGES_REQUESTED** [review]

**Comment by gregfowlerphd** [comment]
@giacomodecolle: Requested changes made.

File: design-patterns/time/CCO DP, p. 17.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ graph TD
     C -->|rdf:type| E[Temporal Instant BFO_0000063]
```
Comment (giacomodecolle) [inline design-patterns/time/CCO DP, p. 17.md:8]: Change BFO number

**Review by giacomodecolle — CHANGES_REQUESTED** [review]

File: design-patterns/time/CCO DP, p. 17.md (hunk 3 above)
Comment (gregfowlerphd) [reply design-patterns/time/CCO DP, p. 17.md]: Done.

**Comment by giacomodecolle** [comment]
Changing name of files to something like "first and last temporal instant"

**Comment by gregfowlerphd** [comment]
-Deleted misnamed Markdown and PNG files
-Uploaded correctly named files
-Content of new files changed to specific examples

**Pull request merged**